_posts/2015-04-06-story-writing-not-a-user.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "Some people are never (ever (never ever)) a user"
+date: 2015-04-06 16:31:00
+---
+
+{% include series/story-writing/header.markdown %}
+
+In the last post in this series, [Rule 4 - Many Users of many types], I gave a
+very loose definition of what I call a user. That was:
+
+>A user is "someone who will use a product of our teams work to do
+something".
+
+Some people this doesn't cover are: your Security Accreditor, your Chief
+Technology Officer, your Single Responsible Officer, the team themselves.
+These people can never ever be users. That's a pretty bold statement but I
+think I can back it up.
+
+##Masquerading Requirements
+
+Lets take the Security Accreditor as an example. Can we really pretend that:
+
+    As a user who has logged in
+    I want my other sessions to be terminated
+    so that no one can impersonate me on another device
+
+is a thing the user wants? Really? Has any user ever said “I really hate when I
+log in to something on two devices and it lets me”?!
+
+That's a Security Accreditor requirement, a risk mitigation, that's being
+masqueraded as a feature and I don't think it should ever be described as a
+User Story with the Security Accreditor as the user.
```

Comment:
You haven't actually described it as a user story with the security accreditor as the user yet. Should take this part out.